src/Storefront/Resources/views/storefront/page/account/order/index.html.twig

```diff
@@ -1,6 +1,6 @@
 {% sw_extends '@Storefront/storefront/page/checkout/confirm/index.html.twig' %}
 
-{% block base_header %}
+{% block base_esi_header %}
```

Comment:
I am not sure if this is a break :thinking:

Reply:
don't think so, you are overwriting/extending a different block and not defining a new one, shouldn't be one 🤔

Reply:
Should be ok if we have already documented that `base_header` was removed from `base.html.twig`